cmd/completion.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright © 2020 The PES Open Source Team pesos@pes.edu
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// completionCmd represents the completion command
+var completionCmd = &cobra.Command{
+	Use:   "completion [bash|zsh|fish|powershell]",
+	Short: "Generate completion script",
+	Long: `To load completions:
+
+Bash:
+
+$ source <(grofer completion bash)
+
+# To load completions for each session, execute once:
+Linux:
+	$ grofer completion bash > /etc/bash_completion.d/grofer 
+MacOS:
+	$ grofer completion bash > /usr/local/etc/bash_completion.d/grofer 
+
+Zsh:
+
+# If shell completion is not already enabled in your environment you will need
+# to enable it.  You can execute the following once:
+
+$ echo "autoload -U compinit; compinit" >> ~/.zshrc
+
+# To load completions for each session, execute once:
+$ yourprogram completion zsh > "${fpath[1]}/_grofer"
```

Comment:
@anihm136 Here's an additional thing you might want to change. I'm assuming `yourprogram` should be `grofer`?

Additionally, this requires documentation updates, so perhaps start with including a section in the README for completions?

Reply:
Yeah, looks like I missed that one. I'll add a section